fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -285,6 +286,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = QUERY_TIMEOUT)
     public int queryTimeoutS = 300;
 
+    @VariableMgr.VarAttr(name = INSERT_TIMEOUT)
+    public int insertTimeoutS = 3000;
```

Comment:
14400s (4h)  might be better, which is in line with broker load timeout (broker_load_default_timeout_second)

Reply:
roger taht